.github/workflows/backup-db.yml

```diff
@@ -0,0 +1,40 @@
+name: Supa-backup
+
+on:
+  push:
+    branches: [ development ]
+  pull_request:
+    branches: [ development ]
+  workflow_dispatch:
+  schedule:
+    - cron: '0 0 * * *'
+
+jobs:   
+  run_db_backup:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    env:
+      supabase_db_url: ${{ secrets.SUPABASE_DB_URL }}
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ref: ${{ github.head_ref }}
```

Comment:
Why do we need to specify the `ref` property here?

Reply:
we don't it was for on pr for qa, fixed now